### DISCUSSION TITLE
[FEATURE]docker macvlan是不是有点问题？

### DISCUSSION THREAD

**Discussion by upn369** [1Panel-dev/1Panel#6932] [Q&A]
### 1Panel 版本

最新

### 请描述您的需求或者改进建议

有IPv6公网地址 所以本地搭建的服务器

发现docker网络的macvlan模式有问题，一开始想着是不是我链接的Wi-Fi有些特殊的原因？后来想想应该不是的，毕竟虚拟机侨接是没问题的，所以应该是你们docker的问题。

还有就是docker容器不能绑定多张网卡。

之所以发现这些问题，主要还是因为是自己本地搭建的。昨天突发奇想想在这个主机里再搭建个openwrt给到我无线路由器上网，然后就发现了不能绑定多网卡，macvlan问题。

建议你们优化下，目前大力提倡IPv6，放开公网IPv6 IP，上行限速，迹象表明以后是允许个人搭建自用服务器的。

**Comment by wanghe-fit2cloud** [comment]
在 1Panel 应用商店部署容器时，默认会使用 `1panel-network` 网络（仅支持 IPv4）。若需启用 IPv6，还需手动进行相关设置。

**Comment by wanghe-fit2cloud** [comment]
也可以看一下这个讨论：https://github.com/1Panel-dev/1Panel/discussions/6766
